oat-sa/tao-item-runner-qti-fe
# Feature/TR-157/Rework the Item Runner's renderer strategies

Related to: https://oat-sa.atlassian.net/browse/TR-157

Rework the way the Item Runner's renderers are loaded.
The main issue was the renderers could only be either `scorer` or the default renderer.
With the changes, we can now redefine the renderers either by:
- give the name of a set of renderers (for the time being `commonRenderer` or `reviewRenderer`)
- give the name of a QTI view (from this [list](https://www.imsglobal.org/question/qtiv2p2p2/QTIv2p2p2-ASI-InformationModelv1p0/imsqtiv2p2p2_asi_v1p0_InfoModelv1p0.html#FigEnumeratedListClass_DataModel_View)), it will be aliased to one of the know renderers
- give any other name, the corresponding renderer will be used if it has been registered, otherwise the default QTI renderers will be used (`commonRenderer`)

The preexisting behavior should not be impacted (say using either the `scorer` or the default renderers).

How to test:
- run the unit tests
- load the branch in `taoQtiItem` and check the items are rendering properly


Impacted unit tests:
- `test/runner/rendererStrategies/test.html`
